
  0%|                                                                                           | 0/109 [00:00<?, ?it/s]































































































































































































































Training Epoch [10/40]:  88%|▉| 96/109 [00:16<00:02,  5.93it/s, acc=0.78760874, dice=0.0053714947, iou=0.0026929798, los


































Training Epoch [12/40]:  90%|▉| 98/109 [00:15<00:01,  7.27it/s, acc=0.7377015, dice=0.0068056467, iou=0.0034144423, loss



































































































Training Epoch [18/40]:  15%|▏| 16/109 [00:02<00:17,  5.24it/s, acc=0.74815106, dice=0.007589123, iou=0.0038090148, loss
Error in callback <function _WandbInit._pause_backend at 0x7ff0581d0280> (for post_run_cell):
Exception in thread NetStatThr:
Traceback (most recent call last):
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 149, in check_network_status
    status_response = self._interface.communicate_network_status()
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 125, in communicate_network_status
    resp = self._communicate_network_status(status)
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 397, in _communicate_network_status
    resp = self._communicate(req, local=True)
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 222, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/share/etud/e2008984/miniconda3/envs/torch_env/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 227, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown